R/checks.R

```diff
@@ -163,8 +163,9 @@ linkify <- function(href, text = shorten(href), title = href, target =
   p_ISI <- function(x) {
     # Web of Science, sökning på UT=WoS-id:
     # http://apps.webofknowledge.com/InboundService.do?product=WOS&action=retrieve&mode=FullRecord&UT=000722432000004
+    # New URL: https://www.webofscience.com/wos/woscc/full-record/WOS:001001819900001
     stringr::str_c(
-      "http://apps.webofknowledge.com/InboundService.do?product=WOS&action=retrieve&mode=FullRecord&UT=",
+      "https://www.webofscience.com/wos/woscc/full-record/WOS:",
```

Comment:
Good change... and it seems like the colon might not require any (https://stackoverflow.com/questions/5330104/encoding-url-query-parameters-in-java/5330261#5330261)[url encoding]